src/main/kotlin/no/nav/familie/ef/iverksett/brev/BrevController.kt

```diff
@@ -20,7 +20,7 @@ class BrevController(
     val journalpostClient: JournalpostClient
 ) {
 
-    @PostMapping("/frittstaaendebrev")
+    @PostMapping("/frittstaende-brev")
```

Comment:
Er det tom kanskje greit å bare kalle den for `frittstaende` då vi allerde har `/api/brev/` i pathen? :P

Reply:
Endret! 405f3e4